custom/workflow-helper/app-deployer/main.go

```diff
@@ -36,17 +37,17 @@ func main() {
 	//GetData is initializing required variables for app-deployer
 	appVars, err := GetData()
 	if err != nil {
-		panic(err.Error())
+		logs.Fatalf("err: %v", err)
```

Comment:
can we have a more detailed log message here?

Reply:
We have been handling it in functions : 
```return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)```

Reply:
return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)

Reply:
[return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)](url)